glide.lock

```diff
@@ -134,7 +134,7 @@ imports:
   subpackages:
   - go
 - name: github.com/prometheus/common
-  version: 195bde7883f7c39ea62b0d92ab7359b5327065cb
+  version: 9e0844febd9e2856f839c9cb974fbd676d1755a8
```

Comment:
It's not in `glide.yaml` so it was bumped when I ran an update. I'll put it in `glide.yaml` to pin it.